### PR TITLE
chore: update IDB DPE repos

### PR DESCRIPTION
**Pull request by kurtisvg**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/sloth/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: src/teams.json
Comment (tmatsuo) [inline src/teams.json:41]: My guess is the line 41 needs a trailing comma.

Comment (kurtisvg) [reply src/teams.json]: Thanks - updated.

**Review by tmatsuo — APPROVED** [review]

**Comment by tmatsuo** [comment]
Waiting for review from @dmahugh

**Review by meredithslota — APPROVED** [review]

**Review by dmahugh — APPROVED** [review]
LGTM

**Pull request merged**
